azurerm/internal/services/netapp/tests/resource_arm_netapp_pool_test.go

```diff
@@ -225,6 +230,10 @@ resource "azurerm_netapp_pool" "test" {
   resource_group_name = azurerm_resource_group.test.name
   service_level       = "Standard"
   size_in_tb          = 15
+
+  tags = {
+    "foo" = "bar"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```